arm-apimanagement/2014-02-14/swagger/service.json

```diff
@@ -0,0 +1,1289 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "APIManagementServiceClient",
+    "description": "REST API for Azure APIManagement Service",
+    "version": "2014-02-14"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "ApiManagement"
+        ],
+        "operationId": "ApiManagement_GetApiManagementServiceOperation",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIManagementServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "type": "file"
```

Comment:
Why do have schema type as file in most of your responses? Will the customer be getting a file? If so then the should the produces be application/json or multipart?